Amethyst/Preferences/GeneralPreferencesViewController.swift

```diff
@@ -176,6 +179,14 @@ final class GeneralPreferencesViewController: NSViewController, CCNPreferencesWi
         floatingTableView.reloadData()
     }
 
+    @IBAction func toggleFloatingLabel(_ sender: NSButton) {
+        updateFloatingLabel()
+    }
+
+    private func updateFloatingLabel() {
+        floatingLabel.stringValue = UserConfiguration.shared.floatingBundleIdentifiersIsBlacklist() ? "Tile Windows:" : "Float Windows:"
```

Comment:
I think you can do this in Interface Builder by binding a transformation of the key in the user defaults controller. That might be more consistent if you can get it to work.

Reply:
Yeah I have something working with a custom transformer. I don't think there are any other custom transformers in the project. Let me know if you want it moved to a different file -- it is pretty specific and probably not going to be used anywhere else, so I kept it in the controller file for now.

Reply:
I followed what was mentioned here to implement the transformer: https://stackoverflow.com/questions/25534334/custom-nsvaluetransformer-in-xcode-6-with-swift